src/js/components/deployments/deployment-report/devicelist.js

```diff
@@ -105,7 +105,7 @@ const deviceListColumns = [
     },
     canShow
   },
-  { key: 'started', title: 'Started', render: ({ device: { created } }) => <MaybeTime value={formatTime(created)} />, sortable: false, canShow },
+  { key: 'started', title: 'Started', render: ({ device: { started } }) => <MaybeTime value={formatTime(started)} />, sortable: false, canShow },
```

Comment:
I tested but it does not work. (lets discuss privately)

Reply:
This fix is correct, let's merge it. We'll follow-up with a dedicated PR once we decide how to update the other labels to make them coherent.